arrow-fx/src/test/kotlin/arrow/fx/ArrowFxSpec.kt

```diff
@@ -1,9 +1,9 @@
 package arrow.fx
 
 import arrow.core.test.UnitSpec
-import io.kotlintest.properties.Gen
-import io.kotlintest.properties.PropertyContext
-import io.kotlintest.properties.forAll
+//import io.kotest.property.Arb
+//import io.kotest.property.PropertyContext
+//import io.kotest.property.forAll
```

Comment:
```suggestion
// import io.kotest.property.Arb
// import io.kotest.property.PropertyContext
// import io.kotest.property.forAll
```

because of Ktlint failures